### PR TITLE
Integration tests with local sdk version

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
Enables a user to specify a local path to their cosmos sdk repository when running integration tests. The validators spawned for integration tests will then reference the local sdk, allowing for better debugging. This implementation is not perfect, but it gets the job done, and unblocks some issues I've been having with debugging integration tests.

### CODE REVIEW

File: integration-tests/testnet-scripts/start-docker.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# Delete old sdk directory if it exists 
+if [ -d "./cosmos-sdk" ]; then
+    rm -rf ./cosmos-sdk/
+fi 
+
+# Copy sdk directory to working dir if path was specified
+if [[ "$LOCAL_SDK_PATH" != "default" ]]
+then
+    cp -n -r "$LOCAL_SDK_PATH" ./cosmos-sdk
```
Comment (smarshall-spitzbart) [inline integration-tests/testnet-scripts/start-docker.sh:24]: This implementation copies the local sdk repo into the ICS repo, pipes it to docker, then deletes the copy. There are alternatives to this scheme, but the chosen solution seemed the simplest and standardizable.

We could also try building the docker image outside the ICS repo to allow ```ADD``` commands with external directories

**Comment by danwt** [comment]
This will be so useful!

File: Dockerfile [hunk 2]
```diff
@@ -13,8 +13,14 @@ WORKDIR /downloads
 # Copy in the repo under test
 ADD . /interchain-security
 
+WORKDIR /interchain-security
```
Comment (jtremback) [inline Dockerfile:16]: Nice

File: integration-tests/testnet-scripts/start-docker.sh (hunk 1 above)
Comment (danwt) [reply integration-tests/testnet-scripts/start-docker.sh:24]: love the `cp -n` 👍

**Review by danwt — APPROVED** [review]

**Pull request merged**